process_report/processors/prepayment_processor.py

```diff
@@ -22,11 +22,24 @@ class PrepaymentProcessor(discount_processor.DiscountProcessor):
     def PREPAY_DEBITS_S3_BACKUP_FILEPATH(self):
         return f"Prepay/Archive/prepay_debits {util.get_iso8601_time()}.csv"
 
+    @property
+    def CREDITS_SNAPSHOT_FILEPATH(self):
+        return f"NERC_Prepaid_Group-Credits-{self.invoice_month}.csv"
+
+    @property
+    def CREDITS_SNAPSHOT_S3_FILEPATH(self):
```

Comment:
While you could argue that these are all constants and therefore should be all caps, please don't.

Reply:
I have repositioned and removed the capitalization on the properties.